third-party-bom/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>net.openhft</groupId>
         <artifactId>root-parent-pom</artifactId>
-        <version>1.2.4</version>
+        <version>1.2.5-SNAPSHOT</version>
```

Comment:
Again, 1.2.4 doesn't seem to be released in nexus. Not sure why its depended on here.